projects_using_gazebo.md

```diff
@@ -4,27 +4,31 @@ This is an incomplete list of projects that use (modern) Gazebo. It includes
 plugins, examples, ROS integrations, etc. If you'd like to add your project,
 please open a pull request by clicking on the "Edit on Github" link.
 
-## Gazebo Plugins
```

Comment:
Do we want to keep the "Gazebo Plugins" section? I was hoping it would encourage people to add items.

Reply:
Sorry, I wasn't opposed to the new "Gazebo tools for Integration" subsection. Just asking if we should still keep the "Gazebo Plugins" subsection as well, even though it's empty.

Reply:
I would personally vote to only do it by project and not delineate the way or method, but let people follow the links to see the integration method instead.

Reply:
I would disagree. More structure (to a reasonable degree) would make it easier to find things. The section below is specifically for Gazebo and ROS 2 integration. If the project is a standalone plugin that implements a specific functionality, e.g. model interaction forces between diple magnets (https://github.com/vustormlab/storm_gazebo_ros_magnet from my grad school days 😉 ), I think it would be nice to highlight it in a separate section since users might be able to use it immediately in their own projects.